contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -136,7 +136,8 @@ static bool check_server_collation_name(char **newval, void **extra, GucSource s
 			* Collation names are stored in lowercase into pg catalog (pg_collation).
 			*/
 		char *dupval = pstrdup(*newval);
```

Comment:
overall, there should be better way to this. We are creating unnecessary two copies just for down casing